test/builtins/Transformers.jl

```diff
@@ -465,6 +465,57 @@ end
     @test infos[:name] == "OneHotEncoder"
     @test infos[:input_scitype] == MLJBase.Table
     @test infos[:output_scitype] == MLJBase.Table
+
+    # test the work on missing values
+    X = (name   = categorical(["Ben", "John", "Mary", "John", missing], ordered=true),
+         height = [1.85, 1.67, 1.5, 1.67],
+         favourite_number = categorical([7, 5, 10, missing, 5]),
+         age    = [23, 23, 14, 23])
+
```

Comment:
This is not a valid table because `name` has a different length from `height`. You need to pad the  outher fields with entries.